contribution_process.md

```diff
@@ -0,0 +1,164 @@
+
+## Contribution-ready stage
+
+A GitHub issue enters the contribution ready stage when a maintainer adds the
+issue to the "Ready to contribute" lane of the public [GitHub project
+board](https://github.com/orgs/aws/projects/21). This stage indicates any user
+is welcome to initiate a contribution via a GitHub pull request.
+
+## Implementation stage
+
+A GitHub issue enters the implementation stage once a user submits a GitHub pull
+request implementing the change. This stage indicates that a contribution has
+been submitted for the issue, but the maintainers have not committed to a full
+review of the contribution. The issue will be moved to the "Implementation" lane
+of the [GitHub project board](https://github.com/orgs/aws/projects/21). The
+maintainers will respond to let the contributor know to expect a preliminary
+review to confirm:
+
+1. It is implemented in the manner described in the issue.
+2. It meets the following  criteria:
+   - The pull request must include passing tests and must pass all existing tests.
+   - The code must work on all supported operating systems.
+   - The code must be documented, which may include inline documentation and user documentation.
+   - A CHANGELOG entry must be provided.
+
+If the criteria are not satisfied, the maintainers will comment on the issue
+specifically indicating what work is still required. If the contributor has
+questions, they should refer to this guide or ask the maintainers to clarify.
+Once the contributor makes the required changes, the maintainers will follow up.
+
+If the pull request is still a work in progress, contributors should open it as a [draft pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request). A preliminary review will not be made on pull requests with a draft status. When the contributor is ready for review, they should mark it as ready (remove draft status). The contributor should feel free to ask questions or request help if their pull request is in the draft status.
+
+If the contributor is no longer actively engaged by responding to comments from
+maintainers or making code changes for more than 30 days, the
+contribution is stale. If not, the pull request will be closed and the
+corresponding issue will be marked as ready for contribution.
+
+Contributions made without a GitHub issue are not prioritized to move to the
+next stage. Pull requests that are not linked to an issue and are more than six
+months old are considered stale and will be closed.
+
+## Ready for review stage
+
+Once the preliminary review criteria from the implementation stage have been
+met, the tracking GitHub issue is labeled as ready for review and added to a
+queue for maintainer review. The maintainers select issues for full review
+from this queue. The issues are prioritized for review publicly based on the
+following criteria in descending order of importance:
```

Comment:
We should reiterate that the priority is publicly viewable from its column in the GitHub board

Reply:
done